lib/rubocop/cop/rails/redundant_presence_validation_on_belongs_to.rb

```diff
@@ -48,12 +48,21 @@ class RedundantPresenceValidationOnBelongsTo < Base
         #
         #   @example source that matches - by a foreign key
         #     validates :user_id, presence: true
+        #
+        #   @example source that DOES NOT match - strict validation
+        #     validates :user_id, strict: true
+        #
+        #   @example source that DOES NOT match - custom strict validation
+        #     validates :user_id, strict: MissingUserError
```

Comment:
Shouldn't this include `presence: true`?

Reply:
Eagle eye. Thanks, fixed.